styles/app.scss

```diff
@@ -1,6 +1,41 @@
 // @import 'yoga';
 // @import 'normalize';
-
 // $fa-font-path: '//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts';
 // @import 'font-awesome';
 @import 'box-sizing';
+@import '_shop-items';
+@import 'variables';
+body {
+
+}
+
+.outer-container {
+    padding: $gutter;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+}
+.container {
+    width: 100%;
+    max-width: $max-content-width;
+    margin: auto;;
+}
+.row {
+    display: flex;
+    flex-flow: wrap;
+
+    @media screen and (max-width: 48rem){
+      display: flex;
+
+      .shop-item {
+        flex-direction: row;
+
+
+      }
+    }
+}
+
+.grid-item {
+  width: calc(100% / 3 - 2rem);
```

Comment:
Set a `min-width` along with `flex-grow` here and your grid should break down nicely for phone sizes without having to add another media query for that break point.